tests/all-examples-from-docs.js

```diff
@@ -1,23 +1,22 @@
 // autogenerated script from all examples in the Chaijs docs https://www.chaijs.com/api/bdd/
-import { describe, expect, chai } from 'https://jslib.k6.io/k6chaijs/4.3.4.1/index.js';
+import chai, {
+  describe,
+  expect
+} from 'https://jslib.k6.io/k6chaijs/4.3.4.1/index.js';
```

Comment:
Not your change, but I think it should use the locally built bin, as other tests do. Also, without this change I get an error: `TypeError: Cannot convert undefined or null to object`
```suggestion
import chai, { describe, expect } from '../build/k6chaijs.min.js';
```